### PR TITLE
change 3 releasing related bugs

### PR DESCRIPTION
**Pull request by alec-z**
1.add license for swagger/servicecomb-kie.yml;
2.add GO_OS choosing function & cp LICENSE NOTICE to the releasing tar.gz  for build_server.sh;
3.add prefix apache_ for the root directory of src.tar.gz in release.sh;

@tianxiaoliang

### CODE REVIEW

File: build/build_server.sh [hunk 1]
```diff
@@ -32,7 +32,13 @@ else
  version=${VERSION}
 fi
 
-
+CURRENT_OS=`go env GOHOSTOS`
+if [ -z "${GO_HOST_OS}" ]; then
+ echo "missing GO_HOST_OS env, use current OS ${GOOS} as host OS"
```
Comment (tianxiaoliang) [inline build/build_server.sh]: if that is build on windows, then "arm_pkg_name" is wrong, is a dead code

**Comment by alec-z** [comment]
@tianxiaoliang  I have change build.sh to build windows, darwin, linux for amd64 at same time. But build arm64 only for linux.

**Comment by alec-z** [comment]
@tianxiaoliang

**Comment by tianxiaoliang** [comment]
the script name is not readable, rename them

**Pull request merged**